packages/react-scripts/config/webpack.config.js

```diff
@@ -523,13 +492,21 @@ module.exports = function(webpackEnv) {
       ],
     },
     plugins: [
+      new ExtensionReloader({
+        entries: {
+          contentScript: ['app', 'contentScript'],
+          background: 'background',
+        },
+      }),
       // Generates an `index.html` file with the <script> injected.
       new HtmlWebpackPlugin(
         Object.assign(
           {},
           {
             inject: true,
-            template: paths.appHtml,
+            template: paths.appPopupHtml,
+            chunks: ['app'],
```

Comment:
Only needs the app linked to the popup because the background and content scripts are loaded via the [`manifest.json`](https://developer.chrome.com/extensions/manifest).